charts/knative/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 apiVersion: v1
 name: knative
+appVersion: 0.7.1
```

Comment:
The patch version number is only relevant to the particular component within Knative. While serving is 0.7.1, Knative as a whole is 0.7.

...then again, Knative build is bundled in the riff chart and the Knative chart only contains serving...

```suggestion
appVersion: 0.7
```